### PR TITLE
Fix Send Reminders; Add New Translations

### PR DESCRIPTION
**Pull request by iamstevedavis**
Need help with Arabic @mahfouzalsheikh if you can... When I try and paste in the template info {{whatever}} it causes the Arabic translations to freak out and change in the IDE. I was using your updated translations from google docs that you posted here:

https://docs.google.com/document/d/1r82HzZqpHGxK5FfiNDEkVHPldbGBcPZXmMrQLUL_7rs/edit?usp=sharing

### CODE REVIEW

File: src/api/handlers/appointments/sendReminders.js [hunk 1]
```diff
@@ -20,8 +20,8 @@ export const sendReminder = async (appointment) => {
     appointmentDateTime: moment(appointment.appointmentTime).format(
       "YYYY-MM-DD h:mm a"
     ),
-    appointmentDate: moment(appointment.appointmentTime).format("YYYY-MM-DD"),
-    appointmentTime: moment(appointment.appointmentTime).format("h:mm a"),
+    appointmentDate: moment(appointment.appointmentTime).format("LL"),
```
Comment (Wiles) [inline src/api/handlers/appointments/sendReminders.js:23]: What are `LL` and `LTS`?

Comment (iamstevedavis) [reply src/api/handlers/appointments/sendReminders.js:23]: LTS : 'h:mm:ss A', LT : 'h:mm A', L : 'MM/DD/YYYY', LL : 'MMMM D, YYYY', LLL : 'MMMM D, YYYY h:mm A', LLLL : 'dddd, MMMM D, YYYY h:mm A' 

I changed it before I seen you had already updated these lines. I'm not sure my changes are correct in terms of how it's supposed to display. For example LTS includes seconds, which are irrelevant. LT might be better.

**Comment by iamstevedavis** [comment]
Does anyone want to review this tonight? I fixed most of @Wiles comment, I can put up a new MR later if he wants me to change it further, no problem. I think he already went to bed though. Different time zone now.

File: src/models/templates.js [hunk 2]
```diff
@@ -4,32 +4,111 @@ const templates = [
   {
     templateId: 1,
     templateName: "Appointment Reminder",
-    English: `Dear {{patientName}}, this message is to inform you of your upcoming appointment{{#description}}: {{description}}{{/description}}.
+    English: `
+      Dear {{patientName}}, this message is to inform you of your upcoming appointment{{#description}}: {{description}}{{/description}}.
```
Comment (mhakmi) [inline src/models/templates.js:8]: Just a minor thing, would be better to remove the `tab` characters preceding each line for templates. Just to reduce the number of chars from the SMS

Comment (mhakmi) [reply src/models/templates.js:8]: This applies to all other lang templates

**Review by mhakmi — APPROVED** [review]
It looks good, just left one minor suggestion. 👍

**Review by mahfouzalsheikh — APPROVED** [review]

**Pull request merged**